axelrod/strategies/graaskamp.py

```diff
@@ -0,0 +1,86 @@
+import random
```

Comment:
Let's put this in the `axelrod_first.py` module: https://github.com/Axelrod-Python/Axelrod/blob/master/axelrod/strategies/axelrod_first.py

Reply:
Yes, totally, didn't see it before, sorry